src/index.js

```diff
@@ -1,12 +1,21 @@
 // Use a Node.js core library
 import * as d3 from 'd3';
+import 'babel-polyfill';
+
 import changeH1Color from './teste';
+import numberPrsNewContributorsAccepted from './15_visualize_number_prs_new_contributors';
 
 // Log the parts object to our browser's console
 d3.select('body').style('background-color', 'lightblue');
 
 console.log(d3.version);
 console.log('testando');
 
-
 changeH1Color();
+
+numberPrsNewContributorsAccepted(
+  'CMU-Perceptual-Computing-Lab',
```

Comment:
create global variables for project name, username and use them here. Other functions may use them too